backend/src/exploregens.ts

```diff
@@ -119,6 +119,7 @@ export class ExploreGens {
     }
 
     private async getFilteredGenerators(query = this.EMPTY, author = this.EMPTY) {
+        query = query || this.EMPTY;
```

Comment:
not needed. var already set at function call.

Reply:
It does not work when null is passed

Reply:
i have also changed the "author" parameter